runatlantis.io/docs/deployment.md

```diff
@@ -66,6 +66,8 @@ If you're using GitLab, navigate to your project's home page in GitLab
 - leave **Enable SSL verification** checked
 - click **Add webhook**
 
+**NOTE:** If you are using Atlantis with multiple GitLab repositories, you must either use the same Integration secret token across all GitLab repositories, or use a separate Atlantis instance for each repository.  The former approach is recommended.**
```

Comment:
I'd think that true with github as well (unless you configure an org level webhook), I might add that if you need to use repos across multiple orgs then it's best to use am instance per org to reduce the impact of a compromise to your other orgs.

Reply:
Yes, I'll merge this and then add the NOTE to the other VCS hosts as well.